### ISSUE TITLE
Panic running command

### ISSUE THREAD

**Issue by jdm** [jdm/libmzx#57]
https://www.digitalmzx.com/show.php?id=413
```
     Running `target/debug/capture image.png output.json unzipped/niterace.mzx`
Capturing board 0:
thread 'main' panicked at 'index out of bounds: the len is 10000 but the index is 10084', /Users/jdm/src/libmzx/src/lib.rs:321:10
stack backtrace:
   0: rust_begin_unwind
             at /rustc/81dc88f88f92ba8ad7465f9cba10c12d3a7b70f1/library/std/src/panicking.rs:475
   1: core::panicking::panic_fmt
             at /rustc/81dc88f88f92ba8ad7465f9cba10c12d3a7b70f1/library/core/src/panicking.rs:85
   2: core::panicking::panic_bounds_check
             at /rustc/81dc88f88f92ba8ad7465f9cba10c12d3a7b70f1/library/core/src/panicking.rs:62
   3: <usize as core::slice::SliceIndex<[T]>>::index
             at /Users/jdm/.rustup/toolchains/nightly-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/slice/mod.rs:3285
   4: core::slice::<impl core::ops::index::Index<I> for [T]>::index
             at /Users/jdm/.rustup/toolchains/nightly-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/slice/mod.rs:3126
   5: <alloc::vec::Vec<T> as core::ops::index::Index<I>>::index
             at /Users/jdm/.rustup/toolchains/nightly-x86_64-apple-darwin/lib/rustlib/src/rust/library/alloc/src/vec.rs:2012
   6: libmzx::Board::level_at
             at /Users/jdm/src/libmzx/src/lib.rs:321
   7: libmzx::robot::run_one_command
             at /Users/jdm/src/libmzx/src/robot.rs:987
   8: libmzx::robot::update_robot
             at /Users/jdm/src/libmzx/src/robot.rs:423
   9: libmzx::board::update_board
             at /Users/jdm/src/libmzx/src/board.rs:187
  10: capture::run
             at ./capture/src/main.rs:141
  11: capture::main
             at ./capture/src/main.rs:220
  12: core::ops::function::FnOnce::call_once
             at /Users/jdm/.rustup/toolchains/nightly-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/ops/function.rs:233
```

**Comment by jdm** [comment]
https://github.com/jdm/libmzx/blob/d604ddedb791c18815f4e422072f89b983c1f167/src/robot.rs#L987

**Comment by jdm** [comment]
Fixed by 615dae20f2d82a2bab6d19cb089ada5225b55d56
